install_config/storage_examples/gluster_backed_registry.adoc

```diff
@@ -62,49 +68,58 @@ NAME            LABELS    STATUS    VOLUME       CAPACITY   ACCESSMODES   AGE
 gluster-claim   <none>    Bound     gluster-pv   1Gi        RWX           24s
 ----
 
-
-
-NOTE: If either the *pvc* or *pv* failed to create or the pvc failed to bind, refer back to the
-link:../persistent_storage/persistent_storage_glusterfs.html[GlusterFS Persistent Storage] guide.
-*Do not* proceed until they initialize and the pvc status is `Bound`
+[NOTE]
+====
+If either the PVC or PV failed to create or the pvc failed to bind, refer back
+to the link:../persistent_storage/persistent_storage_glusterfs.html[GlusterFS
+Persistent Storage] guide. *Do not* proceed until they initialize and the pvc
+status is *Bound*.
+====
 
 [[attach-pvc-to-reg]]
 === Attach the PVC to the Docker Registry
 
-Before moving forward, ensure that the docker-registry service is running.
+Before moving forward, ensure that the *docker-registry* service is running.
 
 ----
 $ oc get svc
 NAME              CLUSTER_IP       EXTERNAL_IP   PORT(S)                 SELECTOR                  AGE
 docker-registry   172.30.167.194   <none>        5000/TCP                docker-registry=default   18m
 ----
 
-NOTE:  If either the docker-registry service or it's associated pod is not running, refer back to the
-link:../install/docker_registry.html#deploy-registry[docker-registry] setup instructions for troubleshooting
-before continuing.
-
+[NOTE]
+====
+If either the *docker-registry* service or its associated pod is not running,
+refer back to the
+link:../install/docker_registry.html#deploy-registry[*docker-registry*] setup
+instructions for troubleshooting before continuing.
+====
 
-Then attach the *pvc*.
+Then, attach the PVC:
 
 ----
 $ oc volume deploymentconfigs/docker-registry --add --name=v1 -t pvc \
      --claim-name=gluster-claim --overwrite
 ----
 
-For further information on utilizing the Docker Registry, navigate to
-link:../install/docker_registry.html[Deploying a Docker Registry].
+link:../install/docker_registry.html[Deploying a Docker Registry] provides more
+information on using the Docker registry.
 
+[[backing-docker-registry-with-glusterfs-storage-issues]]
 == Known Issues
 
+[[backing-docker-registry-with-glusterfs-storage-pod-cannot-resolve]]
 === Pod Cannot Resolve the Volume Host
 
-In non-production cases where the dnsmasq server is located on the same node as the OpenShift master service,
-it is possible that the pods cannot resolve the host machines when mounting the volume,
-causing the `docker-registry-1-deploy` pod to error.
-This can happen when `dnsmasq.service` fails to start because of a collision with OpenShift DNS on port 53.
-To run the DNS server on the master host, some configurations needs to be changed.
+In non-production cases where the *dnsmasq* server is located on the same node
+as the {product-title} master service, it is possible that the pods cannot
+resolve the host machines when mounting the volume, causing the
+*docker-registry-1-deploy* pod to error.This can happen when *dnsmasq.service*
```

Comment:
Is error a verb? Would it be better to say "causing errors in the *docker-registry-1-deploy* pod."?